src/query/interpret/eval.hpp

```diff
@@ -1090,6 +1092,18 @@ class ExpressionEvaluator : public ExpressionVisitor<TypedValue> {
     }
   }
 
+  TypedValue Visit(PatternComprehension &pattern_comprehension) override {
+    TypedValue &frame_pattern_value = frame_->at(symbol_table_->at(pattern_comprehension));
+    if (!frame_pattern_value.IsFunction()) [[unlikely]] {
+      throw QueryRuntimeException(
+          "Unexpected behavior: Exists expected a function, got {}. Please report the problem on GitHub issues",
```

Comment:
here an error occurs with the UNWIND query that I posted below the review. I think you should just throw at this point since you haven't implemented the symbol_generator.cpp and hence frame will not receive the right symbol position.